techniques/ios/MASTG-TECH-0056.md

```diff
@@ -9,9 +9,12 @@ Different methods exist for installing an IPA package onto an iOS device, which
 
 > Please note that iTunes is no longer available in macOS Catalina. If you are using an older version of macOS, iTunes is still available but since iTunes 12.7 it is not possible to install apps.
 
-## Cydia Impactor
+## Sideloadly
```

Comment:
do you mind creating a TOOL file for this and linking it here?

Reply:
Sure. Let's track that as a new  issue in #2657